Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,24 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            string name;
```

Comment:
Declare variables right next to where you use them.

Reply:
This means not declaring all the variables in one place.

Reply:
Thank you for looking this over. Bad habits left over from coding 15 years ago... I've adjusted it following your recommendations.